R/utils.R

```diff
@@ -0,0 +1,124 @@
+#' Custom disconnect message
+#'
+#'
+#' @description
+#' Create the html overlay panel to appear when RSConnect disconnects
+#'
+#' @param refresh the text to appear that will refresh the page when clicked
+#' @param links A list of mirros or alternative links to the dashboard
+#' @param publication_name The parent publication name
+#' @param publication_link The link to the publication on Explore Education Statistics
+#'
+#' @importFrom htmltools tags tagList
+#'
+#'
+#' @return A html overlay panel that appears when RSConnect disconnects for a public R Shiny dashboard in DfE
+#' @export
+#'
+#' @examples
+#' custom_disconnect_message(
+#'   refresh = "Refresh page",
+#'   links = c("https://department-for-education.shinyapps.io/dfe-shiny-template/",
+#'             "https://department-for-education.shinyapps.io/dfe-shiny-template-overflow/"),
+#'   publication_name = "Explore Education Statistics Publication",
+#'   publication_link = "https://explore-education-statistics.service.gov.uk/find-statistics/pupil-attendance-in-schools")
```

Comment:
This line is triggering the note in the R CMD Check, if you make the URL be a new line like this then it will pass without the note. I'd agree with usually ignoring these but given this is doable without breaking the URL we might as well tidy it now:

```
#'   publication_link =
#'   "https://explore-education-statistics.service.gov.uk/find-statistics/pupil-attendance-in-schools"
#'   )
```